src/sidebar/components/selection-tabs.js

```diff
@@ -53,18 +56,13 @@ Tab.propTypes = {
    */
   isWaitingToAnchor: propTypes.bool.isRequired,
   /**
-   * Callback when this tab is active with type as a parameter.
+   * Callback to invoke when this tab is selected.
    */
-  onChangeTab: propTypes.func.isRequired,
+  onSelect: propTypes.func.isRequired,
   /**
    * Is this tab currently selected?
    */
   selected: propTypes.bool.isRequired,
-  /**
-   * The type value for this tab. One of
-   * 'annotation', 'note', or 'orphan'.
-   */
-  type: propTypes.oneOf(['annotation', 'note', 'orphan']).isRequired,
```

Comment:
The `type` prop was only used for the purpose of providing an argument which `onChangeTab` had to be called with. I simplified the props here by simply making the parent component pass an `onSelect` closure that already includes this argument.